Philips.CodeAnalysis.Test/Moq/MockObjectsMustCallExistingConstructorsAnalyzerTest.cs

```diff
@@ -496,6 +496,40 @@ public static void Method()
 			}
 		}
 
+		[DataRow("", false)]
```

Comment:
is this really enough new unit tests for all those if blocks in the analyze method you created?